### PR TITLE
Update theme, title & button components and lib exports

### PR DESCRIPTION
**Pull request by AlinaGoaga**
**What changed?**
- added new exports to lib
- updated theme and title + button components

**Why?**
- need these for EE to ensure styles and layouts are consistent

**How did you test it?**
- browser

### CODE REVIEW

File: ui/components/Button.tsx [hunk 1]
```diff
@@ -18,8 +18,8 @@ export interface Props extends ButtonProps {
 function UnstyledButton({ loading, ...props }: Props) {
   return (
     <MaterialButton
-      variant="outlined"
-      color="primary"
+      variant={props.variant || "outlined"}
+      color={props.color || "primary"}
```
Comment (foot) [inline ui/components/Button.tsx:22]: Could pull this out into `const defaultProps` maybe?

File: ui/components/Icon.tsx [hunk 2]
```diff
@@ -36,6 +46,18 @@ function getIcon(i: IconType) {
...
+      return ArrowUpwardIcon;
+
+    case IconType.DeleteIcon:
+      return DeleteIcon;
+
+    case IconType.NavigateNextIcon:
+      return NavigateNextIcon;
+
+    case IconType.SaveAltIcon:
+      return SaveAltIcon;
```
[the remark this answers is not in the document]
Comment (foot) [reply ui/components/Icon.tsx:59]: `function getIcon(i: IconType | SomeMaterialIconType): SomeMaterialIconType {`

Orrr.. do that more generic parameter adaptation at the `Icon` level?

[the remark this answers is not in the document]
Comment (foot) [reply ui/components/Icon.tsx:59]: Or 

```
function getIcon(i: IconType, notFound = QuesionMarkIcon: elem) {
```

[the remark this answers is not in the document]
Comment (AlinaGoaga) [reply ui/components/Icon.tsx:59]: Yup agreed. Tbh I'd probably just leave at (I am not sure we need the type .. ):

`getIcon(save) {
   case "save":
      return SaveAltIcon;
}`

File: ui/components/Button.tsx (hunk 1 above)
Comment (AlinaGoaga) [reply ui/components/Button.tsx]: Updated

File: ui/components/Icon.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (AlinaGoaga) [reply ui/components/Icon.tsx:59]: Actually we have this exact structure in ReconciliationGraph - there seems to be some duplication. I'll get everything together and remove IconType

File: ui/components/Icon.tsx [hunk 3]
```diff
@@ -36,6 +50,27 @@ function getIcon(i: IconType) {
...
+
+    case IconType.CheckCircleIcon:
+      return CheckCircleIcon;
+
+    case IconType.HourglassFullIcon:
+      return HourglassFullIcon;
+
+    case IconType.ErrorIcon:
+      return ErrorIcon;
+
```
Comment (foot) [inline ui/components/Icon.tsx:73]: ✨

File: ui/pages/Applications.tsx [hunk 4]
```diff
@@ -40,13 +40,16 @@ function Applications({ className }: Props) {
...
+  );
+
   return (
-    <Page className={className}>
-      <Flex align start>
-        <h2>Applications</h2>
-        <Spacer padding="xs" />
-        <PollingIndicator loading={loading} />
-      </Flex>
+    <Page className={className} title={title}>
```
Comment (foot) [inline ui/pages/Applications.tsx:52]:
```suggestion
    <Page className={className}>
      {title}
```

Testing locally these two look the same now after your other styling changes. Could we just do that and skip the conditional logic in `Page.tsx` or is it useful to have for EE too?

Comment (AlinaGoaga) [reply ui/pages/Applications.tsx:52]: Refactored

File: ui/pages/Applications.tsx [hunk 5]
```diff
@@ -40,20 +40,23 @@ function Applications({ className }: Props) {
...
```
Comment (foot) [inline ui/pages/Applications.tsx:52]: Thats works too!

💯

**Review by foot — APPROVED** [review]
✨ ! 🙌

**Pull request merged**